apps/account/tests/test_views/test_passkey_login_view.py

```
@@ -0,0 +1,60 @@
+        response = client.post(
+            reverse("account:passkey-auth-complete"),
+            data={"passkeys": json.dumps({"id": "cred-id", "type": "public-key"})},
+        )
+
+        assert response.status_code == http.HTTPStatus.OK
+        data = json.loads(response.content)
+        assert data["status"] == "OK"
+        assert "redirect" in data
+
+    def test_post_with_invalid_passkey_returns_401(self, client, monkeypatch):
+        monkeypatch.setattr("passkeys.backend.auth_complete", lambda request: None)
```

Ruff (0.15.12)

[warning] Unused lambda argument: `request`

(ARG005)